templates/registration/password_reset_email.html

```diff
@@ -11,8 +11,8 @@ <h1 style="font-size:4em;display:inline;vertical-align: middle"><a href="{{ prot
 <p align="center">
 <a href="{{ protocol }}://{{ domain }}{{ url('password_reset_confirm', uidb64=uid, token=token) }}" style="cursor: pointer;display:block;text-align: center;padding: 4px 2px 5px;color: white;border: 1px solid #666;border-radius: 1px;background: #2980b9;background: linear-gradient(180deg, #00aee0, #2980b9);text-decoration: none;line-height:2em;font-size:1em;width:12em;">{{ _('Reset Password') }}</a>
 </p>
-{% if SITE_ADMIN_EMAIL %}
-{{ _('See you soon! If you have problems resetting your email, feel free to shoot us an email at') }} <a href="mailto:{{ SITE_ADMIN_EMAIL }}">{{ SITE_ADMIN_EMAIL }}</a>
+{% if site_admin_email %}
+{{ _('See you soon! If you have problems resetting your password, feel free to shoot us an email at:') }} <a href="mailto:{{ site_admin_email }}">{{ site_admin_email }}</a>
```

Comment:
I wonder if there are any languages where we don't want the e-mail address at the end. Should we use markdown instead?

Reply:
I used markdown.